roles/openshift-applier/tasks/process-file.yml

```diff
@@ -1,15 +1,14 @@
 ---
 
-- name: "Determine location for the 'file'"
-  vars:
-    path: "{{ file }}"
-  import_tasks: check-file-location.yml
+- name: "Set default values"
```

Comment:
Please make this a bit more descriptive - i.e.: what was there before is probably A_OK